rfcs/20240806-c-api/README.md

```diff
@@ -85,9 +85,8 @@ This assumes that each rank is associated with a device, which has been set befo
 
 | NCCL              | oneCCL (proposed C)          | oneCCL (current, C++)   |
 |-------------------|------------------------------|-------------------------|
-|`ncclResult_t ncclAllgather (sendbuff,recvbuff,count, datatype, op, comm, stream)`|`onecclResult_t onecclAllgather(sendbuff, recvbuff, count, datatype, op, comm, oneccl_stream)`|`ccl::event communicator::allgather (2) (sendbuff, recvbuff, count, datatype, op, comm, oneccl_stream, deps)`|
-|`ncclResult_t ncclAllreduce(sendbuff,recvbuff, count, datatype, op, comm, stream)`|`onecclResult_t onecclAllreduce(sendbuff, recvbuff, count, datatype, op, comm, oneccl_stream)`|`ccl::event 
-communicator::allreduce(sendbuff, recvbuff, count, datatype, op, comm, oneccl_stream, deps)`|
+|`ncclResult_t ncclAllgather (sendbuff,recvbuff,count, datatype, comm, stream)`|`onecclResult_t onecclAllgather(sendbuff, recvbuff, count, datatype, comm, oneccl_stream)`|`ccl::event communicator::allgather (2) (sendbuff, recvbuff, count, datatype, comm, oneccl_stream, deps)`|
+|`ncclResult_t ncclAllreduce(sendbuff,recvbuff, count, datatype, op, comm, stream)`|`onecclResult_t onecclAllreduce(sendbuff, recvbuff, count, datatype, op, comm, oneccl_stream)`|`ccl::event communicator::allreduce(sendbuff, recvbuff, count, datatype, op, comm, oneccl_stream, deps)`|
```

Comment:
nit: formatting

there is no space between `sendbuff,recvbuff,count` but there is space between `count, datatype` and others.

this small issue appears in multiple places in this line

Reply:
fixed